### PR TITLE
Chat data adding. (suggestion)

### PR DESCRIPTION
**Pull request by Aurelien30000**
Adding the using of chat data with CoreProtect-Extension, to complete the data proposed :)

### CODE REVIEW

File: src/main/java/com/djrapitops/extension/CoreProtectExtension.java [hunk 1]
```diff
@@ -47,6 +47,7 @@ of this software and associated documentation files(the "Software"), to deal
     private static final int ACTION_REMOVED = 0;
     private static final int ACTION_PLACED = 1;
     private static final int ACTION_INTERACT = 2;
+    private static final int ACTION_CHAT = 3; // Not sure of the value, need trials.
```
Comment (AuroraLS3) [inline src/main/java/com/djrapitops/extension/CoreProtectExtension.java:50]: Did you manage to test this? :)

**Review by AuroraLS3 — COMMENTED** [review]
Thanks, I didn't realize they had this thing since it wasn't written in their API documentation comment

File: src/main/java/com/djrapitops/extension/CoreProtectExtension.java (hunk 1 above)
Comment (Aurelien30000) [reply src/main/java/com/djrapitops/extension/CoreProtectExtension.java:50]: Not really, I don't have the knowledge to build the plugin :/ Sorry, that's why I'm asking you xD

Comment (Aurelien30000) [reply src/main/java/com/djrapitops/extension/CoreProtectExtension.java:50]: **EDIT** : I found, I'll let you know :)

**Comment by Aurelien30000** [comment]
Sad, impossible after testing :/ Sorry

**Comment by AuroraLS3** [comment]
Aw, A for effort though.

**Pull request closed without merging**